src/Tags/Structure.php

```diff
@@ -135,7 +135,7 @@ public function toArray($tree, $parent = null, $depth = 1)
                 'count'       => $index + 1,
                 'first'       => $index === 0,
                 'last'        => $index === count($tree) - 1,
-                'is_current'  => rtrim($this->currentUrl, '/') === rtrim($url, '/'),
+                'is_current'  => Str::length($url) && $url === $this->currentUrl,
```

Comment:
If you have the homepage (url of `/`) in your nav, it will never be marked as the current now.

Reply:
Thanks!

Reply:
I've added a test for that and fixed it.